packages/mongoose/test/unit/decorators/index.test.ts

```diff
@@ -33,4 +33,33 @@ describe('mgoose decorators', () => {
 			should(propsMetadata).be.deepEqual({ timestamps: true });
 		});
 	});
+
+	describe('@mgoose.virtual()', () => {
+		it('should decorate property correctly', () => {
+			@mgoose.schema({ timestamps: true })
+			class Customer {
+				@mgoose.virtual()
+				@mgoose.prop({ required: false })
+				firstname: string;
```

Comment:
I'm not entirely sure what use case this would cover, as in defining a virtual property with no getter/setter.
But looking at the [mongoose docs](userSchema.virtual('domain')) seems to be supported

Reply:
We used in that way in some APIs. On any case, here I added it just to cover the build of that .ts. 
Before my change that piece of code failed to compile with a ts-error:

TS1236: The return type of a property decorator function must be either 'void' or 'any'.
  Unable to resolve signature of property decorator when called as an expression.
    Type 'TypedPropertyDescriptor<unknown>' is not assignable to type 'void'.